modules/ROOT/pages/coverage-studio-concept.adoc

```diff
@@ -6,51 +6,69 @@ endif::[]
 You can run a coverage report from Anypoint Studio.
 
 The following configurations apply only when you execute your MUnit tests in Studio. They do not apply when running tests from Maven in a CI/CD environment. +
-To run coverage reports from Maven see xref:coverage-maven-concept.adoc[Using Coverage in Maven].
+To run coverage reports from Maven see xref:coverage-maven-concept.adoc[].
 
 == Enable Coverage Reports in Anypoint Studio
 
-The Coverage button in the image bellow allows you to see which flows, and which percentage of event processors in those
-flows were covered by the test:
+The Coverage button enables you to see which flows, and which percentage of event processors in those flows the test covers:
```

Comment:
Consider revising to focus on the user task instead of the UI.

Reply:
Sure! I'll rewrite the focus.